financial-connections/src/main/java/com/stripe/android/financialconnections/features/linkaccountpicker/LinkAccountPickerViewModel.kt

```diff
@@ -377,10 +379,16 @@ internal class LinkAccountPickerViewModel @AssistedInject constructor(
                     )
                     PARTNER_AUTH -> UpdateRequired(
                         generic = genericContent,
-                        type = PartnerAuth(
+                        type = Supportability(
                             institution = institution,
                         ),
                     )
+                    INSTITUTION_PICKER -> UpdateRequired(
+                        generic = genericContent,
+                        type = Supportability(
+                            institution = null,
+                        ),
+                    )
```

Comment:
This is the actual change besides renaming. Treat institution_picker next pane as a supportability case.